### PR TITLE
Ragmala english toggles

### LINKED ISSUE #1224: Clicking in empty space outside of the menus should close the toolbar menu

### LINKED ISSUE #1225: Akhand Paath Bhog Ceremony should have Raag Maalaa Toggle instead of english explanations
There should be two ceremonies in the DB to choose from. The toggle will determine which one is displayed

### LINKED ISSUE #1226: Remove english explanations toggle from Anand Sahib ceremony

### LINKED ISSUE #1229: Clicking on list of banis in sunder gutka does not open up a bani.
While the popular bani and nitnem banis work fine, the extra banis don't have any onclick handler.

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Screenshot 2020-10-19 at 11 53 15 AM
* Fixes #1225 
* Fixes #1226 
* Fixes #1224 
* Fixes #1229 
* Minor code improvements

### CODE REVIEW

File: www/main/common/constants/index.js [hunk 1]
```diff
@@ -5,5 +5,11 @@ export const DEFAULT_OVERLAY = 'none';
 export const nitnemBaniIds = [2, 4, 6, 9, 10, 20, 21, 23];
 export const popularBaniIds = [90, 30, 31, 22];
 export const visibleCeremoniesIds = [1, 3, 5];
```
Comment (tsingh777) [inline www/main/common/constants/index.js:7]: we can remove `visibleCeremoniesIds` now?

File: www/main/addons/ceremonies/components/CeremonyPane.jsx [hunk 2]
```diff
@@ -25,10 +27,21 @@ const CeremonyPane = props => {
     applyTheme(theme);
   };
 
-  const toggleEnglishExplainations = isEnglishExplainations => {
-    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-english`, isEnglishExplainations);
+  const toggleOptions = (toggleType, toggleVar) => {
+    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-${toggleType}`, toggleVar);
     global.platform.updateSettings();
-    loadCeremonyGlobal(id);
+    const ceremonyToLoad = toggleType === 'rm' ? ceremoniesFilter.raagmalaMap[id] : id;
```
Comment (tsingh777) [inline www/main/addons/ceremonies/components/CeremonyPane.jsx:33]: there's a bug with having the toggle off and selecting  a different theme from the ceremony without toggling. The ceremony is loaded with raag mala when the toggle is off.

**Review by tsingh777 — CHANGES_REQUESTED** [review]

File: www/main/addons/ceremonies/components/CeremonyPane.jsx [hunk 3]
```diff
@@ -25,10 +27,22 @@ const CeremonyPane = props => {
...
   };
 
-  const toggleEnglishExplainations = isEnglishExplainations => {
-    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-english`, isEnglishExplainations);
+  const toggleOptions = (toggleType, toggleVar) => {
+    store.setUserPref(`gurbani.ceremonies.ceremony-${token}-${toggleType}`, toggleVar);
     global.platform.updateSettings();
-    loadCeremonyGlobal(id);
+    const ceremonyToLoad =
+      toggleType === 'rm' && !toggleVar ? ceremoniesFilter.raagmalaMap[id] : id;
```
Comment (tsingh777) [inline www/main/addons/ceremonies/components/CeremonyPane.jsx:34]: when toggling, the history is updated. probably shouldn't but the entry in the history for raagmala off is an empty string. 
image

**Review by tsingh777 — CHANGES_REQUESTED** [review]
Still seeing issues when toggle the other themes with raagmala toggle off.

File: www/main/addons/ceremonies/components/CeremonyPane.jsx (hunk 3 above)
Comment (inderpreetsingh) [reply www/main/addons/ceremonies/components/CeremonyPane.jsx:34]: @tsingh777 figured out the problem, the database does not have a Gurmukhi toggle for this ceremony. Preventing addition in history could be the way to go - but that would be pretty hacky as for all intents and purposes on technical side, its a separate ceremony.

Comment (tsingh777) [reply www/main/addons/ceremonies/components/CeremonyPane.jsx:34]: As discussed, we'll fix the DB entry.

Comment (inderpreetsingh) [reply www/main/addons/ceremonies/components/CeremonyPane.jsx:34]: Also it should work now with rm toggle off https://youtu.be/Xd9GrcOJX8Q

**Review by tsingh777 — APPROVED** [review]
LGTM.

**Pull request merged**